larray/core.py

```diff
@@ -7667,26 +7667,67 @@ def opmethod(self, other):
     __ror__ = _binop('ror')
 
     def __matmul__(self, other):
+        """
+        What doc of Numpy matmul says:
```

Comment:
This sentence looks odd, though I have trouble to find a better phrasing. "Quoting the corresponding documentation in Numpy"?